.ci/containers/github-differ/Dockerfile

```diff
@@ -1,6 +1,7 @@
 FROM alpine/git
 
 RUN apk add --no-cache curl
+RUN apk add --no-cache curl-dev
```

Comment:
Why do we need curl and curl-dev?

Reply:
The `curl` package is broken in Alpine and doesn't work without `curl-dev`.  I can't explain why that happened or when it'll be fixed, but it doesn't cost us much.